coc/raid.py

```diff
@@ -149,13 +155,15 @@ def __repr__(self):
             ("district", repr(self.district)),
             ("attacker_tag", self.attacker_tag),
             ("destruction", self.destruction),
+            ("stars", self.stars)
```

Comment:
I usually like to keep trailing commas in because it means if we ever add another line it'll only have 1 line changed on the commit, rather than 2 (1 to add a comma, 1 for the new line)

```suggestion
            ("stars", self.stars),
```